Makefile

```diff
@@ -1,5 +1,20 @@
-.PHONY: clean
+.PHONY: build dev clean
```

Comment:
can we just use package.json scripts instead of also having a makefile?

Reply:
We definitely can, although I wonder if we should move everything into package scripts. For now, I kept the `build` and `clean` targets for historical reasons, but I'd like to hear from you and @kne42 if we should remove the `Makefile` completely.

----

If that's something we want, we could replace the `build` target with a package script that runs `yarn build:prod && jupyter-book build .`, and for the `clean` target, we could use a Gulp task to clean the files:

```js
const del = require('del');
const gulp = require('gulp');

gulp.task('clean', () => {
  return del([
    '_build/html',
    'theme/static/dist',
    'api/stable/**/*.rst',
    '!api/stable/index.rst',
  ]);
});
```

Reply:
don't have a strong opinion there.   partly just asking out of curiosity.  It's always nice to minimize files that accomplish similar tasks if possible, but i can see the argument for leaving it!  happy to defer to @kne42  on that one.

Reply:
yeah, I think we should use the package.json instead of the Makefile now that you have it set up :) seems redundant and also this way it'll work on Windows without requiring a `make.bat` file

Reply:
@kne42 @tlambert03 Sounds good, I went ahead and removed the `Makefile` and replaced them with npm scripts. I also didn't have to write the Gulp task and ended up doing it this way:

```json
    "clean": "run-p clean:*",
    "clean:api-stable": "find api/stable/ ! -name 'index.rst' -type f -exec rm -f {} +",
    "clean:docs": "rm -rf _build/html",
    "clean:theme": "rm -rf theme/napari/static/dist",
```

Probably not much performance difference, but now we're cleaning files in parallel 😆